packages/kbn-pm/src/commands/bootstrap.ts

```diff
@@ -73,6 +74,15 @@ export const BootstrapCommand: ICommand = {
       });
     }
 
+    if (process.platform === 'darwin' && process.arch === 'arm64') {
+      const patchNativeModulesStartTime = Date.now();
+      await patchNativeModulesForArmMacs(log, kibanaProjectPath);
+      timings.push({
+        id: 'patch native modudles for arm macs',
+        ms: Date.now() - patchNativeModulesStartTime,
+      });
+    }
```

Comment:
@mistic I placed this here because it's after the `--force-install` run of yarn, but when we don't need a force install this happens before bazel potentially updates the `node_modules` before running the package builds. At this moment we don't actually need either of the two modules updated here in package builds, so I should probably just move this after `//packages:build`, but do you think there's a way we could use a babel run command or something to execute this logic after the specific node modules are updated?